code/game/objects/effects/effect_system/effects_other.dm

```diff
@@ -134,4 +134,57 @@
 				L.Knockdown(rand(20,100))
 		return
 	else
-		dyn_explosion(location, amount, flashing_factor)
\ No newline at end of file
+		dyn_explosion(location, amount, flashing_factor)
+
+/datum/effect_system/trail_follow/ion/space_trail
+	var/turf/oldloc // secondary ion trail loc
+	var/turf/currloc
+
+/datum/effect_system/trail_follow/ion/space_trail/Destroy()
+	oldloc = null
+	currloc = null
+	return ..()
+
+/datum/effect_system/trail_follow/ion/space_trail/start() //fuck whoever put 20 src's in here making me remove them
+	if(!on)
+		on = 1
+		processing = 1
+	if(processing)
+		processing = 0
+		spawn(0)
```

Comment:
see above about ```spawn()``` (```spawn(0)```, like ```spawn()```, can be replaced with ```INVOKE_ASYNC()```)
on that note, fix any other spawns in the PR too.